acceptable/_service.py

```diff
@@ -13,7 +19,7 @@ class InvalidAPI(Exception):
     pass
 
 
-class APIMetadata:
+class APIMetadata(object):
```

Comment:
IME `__metaclass__ = type` at the top is nicer than having to do this everywhere.